src/worker/utils/initRDKit.ts

```diff
@@ -25,16 +25,22 @@
 import { MAX_CACHED_JSMOLS } from '../../constants';
 import { RDKitProviderCacheOptions } from '../../contexts';
 
-export const initRdkit = async ({ preferCoordgen, removeHs, cache = {} }: InitWorkerOptions) => {
+export const initRdkit = async ({ rdkitPath, preferCoordgen, removeHs, cache = {} }: InitWorkerOptions) => {
   if (cache) {
     initWorkerCache({ cache, removeHs });
   }
+
+  if (globalThis.workerRDKit) return;
+
+  const path = rdkitPath || '/RDKit_minimal.js';
+  const url = new URL(path, globalThis.origin);
   //@ts-ignore
-  importScripts(`${globalThis.origin}/RDKit_minimal.js`);
+  importScripts(url);
   if (!globalThis.initRDKitModule) return;
   await globalThis.initRDKitModule().then((rdkitModule) => {
     globalThis.workerRDKit = rdkitModule;
   });
+  //@ts-ignore
```

Comment:
What was wrong with ts here?

Reply:
TS error: `prefer_coordgen` not existing on type `never`.
Changed the function to avoid it.